test/cli_test.py

```diff
@@ -200,6 +200,25 @@ def test_invalid_link_type(self):
         with pytest.raises(ConfigValueError):
             self.runcli("alt", "update", "by-year")
 
+    def test_album_art_linked(self, tmp_path: Path):
+        self.alt_config["album_art_copy"] = True
+        self.config["art_filename"] = "COVER"
+        album = self.add_album(
+            artist="Michael Jackson",
+            album="Thriller",
+            year="1990",
+            original_year="1982",
+        )
+        album.set_art(self.IMAGE_FIXTURE1)
+        album.store()
+        self.runcli("alt", "update", "by-year")
+
+        external_album_path = tmp_path / "beets_lib" / "by-year" / "1990" / "Thriller"
+        external_art_path = external_album_path / "COVER.png"
+
+        # Symlink is created
+        assert external_art_path.is_symlink()
```

Comment:
Let’s also check that the symlink points to the album art

```suggestion
        assert_symlink(external_art_path, Path(str(album.artpath, "utf8")))
```